### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by clem21**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,15 +24,28 @@ This plugin gives you the ability to have Plugins oriented settings in your favo
 
 **⚠️ This plugin is not released yet.**
```
Comment (jacquesbh) [inline README.md:25]: Actually it is!

File: README.md [hunk 2]
```diff
@@ -24,15 +24,28 @@ This plugin gives you the ability to have Plugins oriented settings in your favo
...
-- Run the diff in your migrations (`./bin/console doctrine:migration:diff`).
-- Execute the migrations (`./bin/console doctrine:migration:migrate`).
+- Require the plugin via composer 
+  ```bash
+  composer require monsieurbiz/sylius-settings-plugin="@rc" --no-scripts
+  ```
+- Edit the `config/bundles.php` to add this line
+    ```bash  
+    MonsieurBiz\SyliusSettingsPlugin\MonsieurBizSyliusSettingsPlugin::class => ['all' => true],
+    ```
```
Comment (jacquesbh) [inline README.md:36]: The indentation looks incoherent with the other steps.

**Review by jacquesbh — APPROVED** [review]
Just a few changes if you can.

**Review by jacquesbh — APPROVED** [review]

**Review by maximehuran — APPROVED** [review]

**Comment by jacquesbh** [comment]
Thank you @clem21! 👋

**Pull request merged**